plio/queries.py

```diff
@@ -18,45 +18,60 @@ def get_plio_details_query(plio_uuid: str, schema: str):
         WHERE plio.uuid  = '{plio_uuid}'"""
 
 
-def get_sessions_dump_query(plio_uuid: str, schema: str):
+def get_sessions_dump_query(plio_uuid: str, schema: str, extra_data: dict):
     """Returns the dump of all the sessions for the given plio"""
```

Comment:
can we also update the docstring with the parameters and mention the dict elements this function expects?